test/test_trixi2img.jl

```diff
@@ -4,7 +4,7 @@ using Trixi
 using Trixi2Img
 
 # pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
-const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples")
+const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
```

Comment:
```suggestion
const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples")
```

Reply:
The additional `"2d"` is added via a `joinpath` in https://github.com/trixi-framework/Trixi2Img.jl/blob/75a8e42dc336be7272702502b395b9e3b269dc1d/test/test_2d.jl#L14

Reply:
Fixed in 71f521b